pages.es/common/7zr.md

```diff
@@ -27,3 +27,7 @@
 - [l]ista el contenido de un archivo:
 
 `7zr l {{ruta/al/archivo.7z}}`
+
+- Establece el nivel de compresión (más alto significa más compresión, pero más lento):
+
+`7zr a {{ruta/al/archivo.7z}} -mx={{0|1|3|5|7|9}} {{ruta/al/archivo_o_directorio}}`
```

Comment:
On PR #14307 , on the second path, you have used `ruta/a/archivo_o_directorio`, here it is `ruta/al/archivo_o_directorio`. 
Can you please keep a pattern? I mean, either change the other PR to `al`, or this to `a`.

Reply:
done